uint8 SUPPLY_PRESSURE # UINT8 Length 1 rw
uint8 EJECTOR_CONTROL # UINT8 Length 16 rw
uint8 SETPOINT_H1 # UINT16 Length 16 x 2 rw
uint8 HYSTERESIS_h1 # UINT16 Lenth 16 x 2 rw
uint8 SETPOINT_H2 # UINT16 Lenth 16 x 2 rw
uint8 HYSTERESIS_h2 # UINT16 Lenth 16 x 2 rw